lib/particular-calendars/france.coutances.ts

```diff
@@ -0,0 +1,244 @@
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marthe-le-bouteiller-vierge
+    marthe_aimee_adele_le_bouteiller_virgin: {
+      dateDef: { month: 3, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    paterne_of_avranches_bishop_and_scubilion_abbot: {
+      dateDef: { month: 4, date: 16 },
+      martyrology: ['paterne_of_avranches_bishop', 'scubilion_abbot'],
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    william_firmatus_abbot: {
+      dateDef: { month: 4, date: 24 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    louis_grignion_de_montfort_priest: {},
+
+    // src: mr_fr_1982_ed2_coutances
+    achard_of_saint_victor_bishop: {
+      dateDef: { month: 4, date: 29 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    marculf_abbot: {
+      dateDef: { month: 5, date: 4 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    germanus_of_normandy_bishop: {
+      dateDef: { month: 5, date: 6 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marie-catherine-de-saint-augustin-vierge
+    catherine_of_saint_augustine_de_simon_de_longpre_virgin: {
+      dateDef: { month: 5, date: 8 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    ortaire_de_landelles_abbot: {
+      dateDef: { month: 5, date: 21 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    severus_of_avranches_bishop: {
+      dateDef: { month: 7, date: 6 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    thomas_becket_bishop: {
+      dateDef: { month: 7, date: 7 },
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    dedication_of_the_cathedral_of_coutances_france: {
+      dateDef: { month: 7, date: 12 },
+      precedence: Precedences.ProperFeast_DedicationOfTheCathedralChurch_8b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    helier_of_jersey_martyr: {
+      dateDef: { month: 7, date: 16 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    marie_madeleine_postel_virgin: {
+      dateDef: { month: 7, date: 17 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    clair_of_normandy_priest: {
+      dateDef: { month: 7, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    john_eudes_priest: {},
```

Comment:
While `john_eudes_priest` is inscribed into the [GRC](https://github.com/romcal/romcal/blob/3c8fee93243ee5b71c59b3ca2e9b30fe07a2c604/lib/general-calendar/proper-of-saints.ts#L761-L764), here the rank is raised to a `memory`.

```suggestion
    john_eudes_priest: {
        precedence: Precedences.ProperMemorial_11b,
    },
```